File: C_Assignment/Assignment_14/programAQ2.c
///////////////////////////////////////////////////////////
//
//  File name : Assigment14Question2.c
//  Descreption : Accept number of rows and number of columns from user and display below pattern
//  Author : Kartik Ganesh Jare
//  Date : 26/5/25
//  
///////////////////////////////////////////////////////////

/* Input : iRow = 4 iCol = 4
Output : 2 4 6 8 10
         1 3 5 7 9 
         2 4 6 8 10
         1 3 5 7 9  
*/

#include<stdio.h>

void Pattern(int iRow,int iCol)
{   
    int i = 0,j = 0,iNo1 = 0,iNo2 = 0;

    for(i = 1 ; i <= iRow; i++)
    {
        for(j = 1,iNo1 = 1, iNo2 = 2; j <= iCol; j++,iNo1 += 2,iNo2 += 2)
        {
            if(i % 2 == 0)
            {
                printf("%d\t",iNo1);
            }
            else
            {
                printf("%d\t",iNo2);
            }
        }
        printf("\n");
    }
        
} 

int main()
{
    int iValue1 = 0, iValue2 = 0; 
    
    printf("Enter number of rows and columns : \n");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}
    

File: Java_Assignment/Assignment_46/programAQ2.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement46Question2.java
//  Description : Write Java program which accept number of rows and number of columns
//                from user and display below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*
Input : iRow = 4 iCol = 4
Output : A B C D
a b c d
A B C D
a b c d

Program Layout :
*/

import java.util.Scanner;

class Pattern
{
    public void PatternX(int iRow,int iCol)
    {   
        int i = 0, j = 0;

        for(i = 1;i <=iRow;i++)
        {
            for(j = 1;j <= iCol;j++)
            {
                if((i % 2) != 0)
                {
                    System.out.print((char)(96 + j) + "\t"); 
                }
                else
                {
                    System.out.print((char)(64 + j) + "\t");
                }
            }
            System.out.println();
        }
    }
}

public class programAQ2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of Row: ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of Columns: ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.PatternX(iValue1, iValue2);

        sobj.close();
    }   
}